tests/cli/dorq/test_entrypoint.py

```diff
@@ -276,3 +276,27 @@ def test_with_flag(self, entrypoint, mock_login_action, flag, expected_runtime):
         mock_login_action.assert_called_with(
             config=ANY, url=ANY, token=ANY, runtime_name=expected_runtime
         )
+
+
+class TestVersion:
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏We don't have any test for the functionality of the version flag. Can you add a test that checks that the expected output is printed?

Reply:
[f78052b](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/252/commits/f78052bab0085849c0cac7fdd981ac8d743c423f)